sdk/android/skin/src/main/java/com/ooyala/android/skin/BridgeMessageBuilder.java

```diff
@@ -340,10 +340,31 @@ public static WritableMap buildMultiAudioParams(List<AudioTrack> audioTracks) {
     return params;
   }
 
+
+  /**
+   * @param castMediaRoutes The set of available devices for casting.
+   * @return list available devices.
+   */
+  public static WritableMap buildCastDeviceListParams(Object castMediaRoutes) {
+    WritableMap params = Arguments.createMap();
+    WritableArray castDeviceNames = Arguments.createArray();
+    WritableArray castDeviceIds = Arguments.createArray();
+
```

Comment:
Init `castDeviceNames ` and `castDeviceIds` into `if (castMediaRoutes instanceof Set)` block

Reply:
it is bad idea, in my point of view. Because in my variant if castMediaRoutes is null or incorrect - we pass empty lists in skin. In your variant we pass nulls.